src/test/java/org/opensearch/security/InitializationIntegrationTests.java

```diff
@@ -178,7 +178,7 @@ public void testWhoAmIForceHttp1() throws Exception {
             Response whoAmIRes = restHighLevelClient.getLowLevelClient().performRequest(new Request("GET", "/_plugins/_security/whoami"));
             assertThat(200, is(whoAmIRes.getStatusLine().getStatusCode()));
             // The HTTP/1.1 is forced and should be used instead
-            assertThat(HttpVersion.HTTP_1_1, is(whoAmIRes.getStatusLine().getProtocolVersion()));
+            assertThat(whoAmIRes.getStatusLine().getProtocolVersion(), is(HttpVersion.HTTP_1_1));
```

Comment:
The order of args was wrong sadly: <actual>, <expected>:

```
public static <T> void assertThat(T actual, Matcher<? super T> matcher) {
```

Fixing just one place but there are many more :(